metricbeat/module/redis/info/data.go

```diff
@@ -27,8 +27,10 @@ var (
 	schema = s.Schema{
 		"clients": s.Object{
 			"connected":           c.Int("connected_clients"),
-			"longest_output_list": c.Int("client_longest_output_list"),
-			"biggest_input_buf":   c.Int("client_biggest_input_buf"),
+			"longest_output_list": c.Int("client_longest_output_list", s.Optional), // Redis 4.x and below
+			"biggest_input_buf":   c.Int("client_biggest_input_buf", s.Optional), // Redis 4.x and below
+			"max_output_buffer":   c.Int("client_recent_max_output_buffer", s.Optional), // Redis 5.0 and above
+			"max_input_buffer":    c.Int("client_recent_max_input_buffer", s.Optional), // Redis 5.0 and above
```

Comment:
Test build is complaining about format here, could you fix it with `gofmt -w metricbeat/module/redis/`?

Reply:
Done.